myRental/logs/Error.log

```diff
@@ -0,0 +1 @@
+Fri Feb 09 2018 20:36:42 GMT-0600 (CST) - LOG::: | ERROR ᑀ Test
```

Comment:
Do you have to include this file, or did it sneak in before you gitignored it?

Reply:
Unfortunately I need at least a blank file.